src/julia.jl

```diff
@@ -156,7 +158,7 @@ function build_julia(_repo_path::String, config::Configuration)
 
     # build and install Julia
     install_dir = mktempdir()
-    build_config = Configuration(; rootfs="package_linux", xvfb=false)
+    build_config = Configuration(; rootfs="package_linux.x86_64", xvfb=false)
```

Comment:
Why add an aarch rootfs artifact while hard-coding use of the x86 one?

Reply:
I assumed there was a need to manually control the rootfs image given this kwarg exists. Are you suggesting removing this kwarg or automatically appending the arch to the name?